mcs/class/System/Test/System.Net/HttpListener2Test.cs

```diff
@@ -62,23 +62,55 @@ public Socket GetSocket ()
 			}
 		}
 
-		public static HttpListener CreateAndStartListener (string prefix)
+		public static HttpListener CreateAndStartListener (string host, int port, string path, AuthenticationSchemes? authSchemes = null, Action<HttpListener> initializer = null)
```

Comment:
please move this and the other methods to `NetworkHelpers` so unrelated test classes don't need to call into `HttpListener2Test`.

This will also fix the Android SDK build since for some reason we exclude HttpListener2Test.cs in the monodroid profile.

Reply:
@akoeplinger thanks, I've changed it as requested.